ui/src/plugins/dev.perfetto.RecordTraceV2/index.ts

```diff
@@ -34,11 +34,19 @@ import {TracedWebsocketTargetProvider} from './traced_over_websocket/traced_webs
 import {savedConfigsPage} from './pages/saved_configs';
 import {WebDeviceProxyTargetProvider} from './adb/web_device_proxy/wdp_target_provider';
 import m from 'mithril';
+import {LOCAL_ABSTRACT_SOCKET_FLAG} from './local_flag';
 export default class implements PerfettoPlugin {
   static readonly id = 'dev.perfetto.RecordTraceV2';
   private static recordingMgr?: RecordingManager;
 
   static onActivate(app: App) {
+    LOCAL_ABSTRACT_SOCKET_FLAG.flag = app.featureFlags.register({
```

Comment:
IMO this should be a user setting not a feature flag. Settings are the right place for user controllable behaviour which is not an implementation detail of the UI.

We used to abuse flags for that but now we have settings, it should live there.